components/lib/crash/src/main/java/mozilla/components/lib/crash/service/GleanCrashReporterService.kt

```diff
@@ -132,6 +144,9 @@ class GleanCrashReporterService(
             when (line) {
                 UNCAUGHT_EXCEPTION_KEY -> ++uncaughtExceptionCount
                 CAUGHT_EXCEPTION_KEY -> ++caughtExceptionCount
+                MAIN_PROCESS_NATIVE_CODE_CRASH_KEY -> ++mainProcessNativeCodeCrashCount
+                FOREGROUND_CHILD_PROCESS_NATIVE_CODE_CRASH_KEY -> ++foregroundChildProcessNativeCodeCrashCount
+                BACKGROUND_CHILD_PROCESS_NATIVE_CODE_CRASH_KEY -> ++backgroundChildProcessNativeCodeCrashCount
                 FATAL_NATIVE_CODE_CRASH_KEY -> ++fatalNativeCodeCrashCount
```

Comment:
Should these be removed?

Reply:
@Dexterp37 suggested [here](https://github.com/mozilla-mobile/android-components/issues/11876#issuecomment-1070530755) that we keep the deprecated keys around for a short while so that the pending data can still be submitted correctly.

Reply:
Understood.  Please create an issue to track removing the deprecated keys.  Thanks

Reply:
Will do